includes/REST_API/Publisher_Logos_Controller.php

```diff
@@ -183,7 +183,9 @@ public function update_item_permissions_check( $request ) {
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_items( $request ) {
-		$publisher_logos = $this->filter_publisher_logos( (array) $this->settings->get_setting( $this->settings::SETTING_NAME_PUBLISHER_LOGOS, [] ) );
+		$publisher_logos_ids = (array) $this->settings->get_setting( $this->settings::SETTING_NAME_PUBLISHER_LOGOS, [] );
```

Comment:
Nit: alternatively, could add a phpdoc block with more precise typing

```suggestion
		/**
		 * Publisher logo IDs.
		 *
		 * @var int[] $publisher_logos_ids
		 */
		$publisher_logos_ids = $this->settings->get_setting( $this->settings::SETTING_NAME_PUBLISHER_LOGOS, [] );
```